hub/management/commands/import_policing_areas.py

```diff
@@ -0,0 +1,241 @@
+        )
+        if scottish_las.exists():
+            pfa_to_lads["S92000003"] = [
+                {"area": la, "pfa_name": "Scotland", "pfa_code": "S92000003"}
+                for la in scottish_las
+            ]
+            if diagnostics:
+                print(f"Found {scottish_las.count()} Scottish local authorities")
+
+        # Add Northern Ireland - all NI local authorities
+        ni_las = Area.objects.filter(
+            gss__startswith="N", area_type__code__in=["STC", "DIS"]
+        )
+        if ni_las.exists():
+            pfa_to_lads["N92000002"] = [
+                {
+                    "area": la,
+                    "pfa_name": "Northern Ireland",
+                    "pfa_code": "N92000002",
+                }
+                for la in ni_las
+            ]
+            if diagnostics:
+                print(f"Found {ni_las.count()} Northern Irish local authorities")
+
+        if diagnostics or not quiet:
+            print(
+                f"Importing {len(pfa_to_lads)} policing areas (England, Wales, Scotland, NI)..."
+            )
+
+        disable = quiet or diagnostics
+        for pfa_code, lad_list in tqdm(list(pfa_to_lads.items()), disable=disable):
+            pfa_name = lad_list[0]["pfa_name"]
+
+            if diagnostics:
+                print(f"\nProcessing {pfa_name} ({pfa_code})")
+                print(f"  Contains {len(lad_list)} local authorities")
+
+            # Get all constituent local authority areas
+            geometries = []
+            la_areas = []
+
+            for lad in lad_list:
+                # For Scotland/NI, we already have the Area object
+                if "area" in lad:
+                    la_area = lad["area"]
+                else:
+                    # For England/Wales, look up by GSS code
+                    lad_code = lad["lad_code"]
+                    try:
+                        la_area = Area.objects.get(
+                            gss=lad_code, area_type__code__in=["STC", "DIS"]
+                        )
+                    except Area.DoesNotExist:
+                        if diagnostics:
+                            print(f"  Could not find LA with GSS {lad_code}")
+                        continue
+
+                la_areas.append(la_area)
+
+                # Extract geometry
+                if la_area.geometry:
+                    geojson = json.loads(la_area.geometry)
+                    # Extract the geometry part from the Feature
+                    if geojson.get("type") == "Feature":
+                        geom = shape(geojson["geometry"])
+                    else:
+                        geom = shape(geojson)
+                    geometries.append(geom)
+                elif diagnostics:
+                    print(f"  LA {la_area.name} has no geometry")
+
+            if not geometries:
+                if diagnostics:
+                    print(f"  No geometries found for {pfa_name}, skipping")
+                continue
```

Comment:
again, I think if we can't find a geometry for a PCC that should always be printed out